lib/api/v1/forms/keywords.go

```diff
@@ -0,0 +1,79 @@
+package forms
+
+import (
+	"encoding/csv"
+	"errors"
+	"io"
+	"mime/multipart"
+
+	"github.com/markgravity/golang-ic/database"
+	"github.com/markgravity/golang-ic/lib/jobs"
+	"github.com/markgravity/golang-ic/lib/models"
+
+	"github.com/fatih/structs"
+)
+
+type KeywordsForm struct {
+	FileHeader *multipart.FileHeader `form:"file" binding:"required"`
+	User       *models.User
+}
+
+func (f *KeywordsForm) Save() error {
+	keywords, err := f.createKeywordsFromCSVFile()
+	if err != nil {
+		return err
+	}
+
+	for _, k := range keywords {
+		job := jobs.Crawl{}
+		job.SetArgs(structs.Map(k))
+		err = jobs.Dispatch(&job)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (f *KeywordsForm) createKeywordsFromCSVFile() ([]models.Keyword, error) {
+	if f.FileHeader.Header.Get("Content-Type") != "text/csv" {
+		return nil, errors.New("file type is not supported")
+	}
+
+	file, err := f.FileHeader.Open()
```

Comment:
Do we need to close the file at some point? I remember there is a convenient way to do that in Go (something like `defer out.Close()`). Can you have a look?